features/support/specialist_sector_helper.rb

```diff
@@ -1,7 +1,19 @@
 require 'gds_api/test_helpers/content_api'
+require "gds_api/test_helpers/content_store"
+
+Before do
```

Comment:
I honestly didn't know a capitalize `before` would work, even though (sorry I know it's nitpicking) could you change it to lowercase please ?